devel/bamtools/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        pezmaster31 bamtools 2.5.1 v
+
+categories          devel
+license             MIT
+platforms           darwin
+maintainers         nomaintainer
+
+description         a programmer's API and an end-user's toolkit for handling BAM files
+long_description    BamTools provides both {*}${description}.
+
+checksums           rmd160  a537678aaad8ea323f7d8f86d92983353220d0fc \
+                    sha256  a933067515bd36ee6edea2379bc156a38d54c67e4c11243eea6cb56a1ebb1ae3 \
+                    size    549902
+
+depends_lib-append  port:jsoncpp
+
+compiler.cxx_standard   2011
```

Comment:
The port fails to build, with various C++11-related warnings and errors. Based on this line, you're ensuring that a C++11-capable compiler is present, but I don't see something like `-std=c++11` in the CXXFLAGS which would be required to guarantee that the compiler is in C++11 mode. You may just need:
```
configure.cxxflags-append -std=c++11
```
You should also [add a comment](https://lists.macports.org/pipermail/macports-dev/2020-August/042287.html) that the C++11 requirement is being imposed by jsoncpp.

Reply:
I added that, and my build here only resulted in 2 warnings, no errors.

```
:info:build /opt/local/libexec/llvm-9.0/bin/../include/c++/v1/set:1127:30: warning: the specified comparator type does not provide a viable const call operator [-Wuser-defined-warnings]
:info:build         static_assert(sizeof(__diagnose_non_const_comparator<_Key, _Compare>()), "");
:info:build                              ^
```